client/components/recycleAndReuseComponents/VerifyItem.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import { Alert, AlertIcon, Box, Button, Checkbox, CloseButton, Flex, HStack, ListItem, Select, Spacer, Text, UnorderedList, VStack } from '@chakra-ui/react'
+import Image from 'next/image'
+import { useRouter } from 'next/dist/client/router'
+
+const VerifyItem = ({ items, setItems, navigateToTakeAction }) => {
+    const router = useRouter()
+
+    // TODO: figure out how this is populated
+    const getInvalidItems = (items) => {
+        return ["Cheese", "Cheeseburger"]
+    }
```

Comment:
I don't really understand the part about whether the item is in the database or not. Maybe some clarifications would help 🤔 For now, hardcoding some values to see the alert.